docs/get-started/getting-started.md

```diff
@@ -417,11 +417,14 @@ With `nv_peer_mem`, IB/RoCE-based transfers can perform zero-copy transfers dire
 2) Install [UCX 1.8.1](https://github.com/openucx/ucx/releases/tag/v1.8.1). 
 
 3) You will need to configure your spark job with extra settings for UCX (we are looking to 
-simplify these settings in the near future):
+simplify these settings in the near future). Choose the version of the shuffle manager
+that matches your Spark version. Currently we support
+Spark 3.0 (com.nvidia.spark.rapids.spark30.RapidsShuffleManager) and
+Spark 3.1 (com.nvidia.spark.rapids.spark31.RapidsShuffleManager):
```

Comment:
Apache Spark really broke the shuffle manager API in a minor version? 😞

Reply:
it;s not minor version 3.0 to 3.1, plus shufflemanager api is officially private.

Reply:
Bummer.  Wish we could setup the shuffle manager via the main plugin so users don't have to get out the decoder ring when configuring the shuffle.

Reply:
sorry guess you mean FEATURE version, I was thinking maintenance release. But its private right now anyway.

Reply:
The main issue is whether the executors need the shuffle manager defined before the SQL plugin gets instantiated I believe. I'd need to do some testing, but seems like it should be a follow up issue.